JustSaying.AwsTools/MessageHandling/SqsNotificationListener.cs

```diff
@@ -88,34 +88,32 @@ public void Listen()
                         await ListenLoop(_cts.Token);
                     }
                 })
-                .ContinueWith(t =>
-                    {
-                        if (t.IsCompleted)
-                        {
-                            Log.Info(
-                                "[Completed] Stopped Listening - {0}", 
-                                queueInfo);
-                        }
-                        else if (t.IsFaulted)
-                        {
-                            Log.Info(
-                                "[Failed] Stopped Listening - {0}\n{1}", 
-                                queueInfo, 
-                                t.Exception);
-                        }
-                        else
-                        {
-                            Log.Info(
-                                "[Canceled] Stopped Listening - {0}", 
-                                queueInfo);
-                        }
-                    });
+                .ContinueWith(t => LogTaskEndState(t, queueInfo));
 
             Log.Info(
                 "Starting Listening - {0}", 
                 queueInfo);
         }
 
+        private static void LogTaskEndState(Task task, string queueInfo)
+        {
+            if (task.IsFaulted)
+            {
+                Log.Info(
+                    "[Faulted] Stopped Listening - {0}\n{1}",
+                     queueInfo,
+                     task.Exception);
+            }
+            else
+            {
+                var endState = task.Status.ToString();
+                Log.Info(
+                    "[{0}] Stopped Listening - {1}",
```

Comment:
Is this particular log message format important? I would think that "Queue Foo Stopped in state Completed" would be better than "Completed - Stopped listening [Foo]"